ci/da-ghc-lib/compile.yml

```
@@ -11,7 +11,7 @@ jobs:
   variables:
     ghc-lib-sha: '77fcc42eef2256ba9bb61a7e442a65bedb19a35d'
     base-sha: '9c787d4d24f2b515934c8503ee2bbd7cfac4da20'
-    patches: '323dc6fcb127f77c08423873efc0a088c071440a 833ca63be2ab14871874ccb6974921e8952802e9'
+    patches: '323dc6fcb127f77c08423873efc0a088c071440a ce802bbf81e25f919381e10b0e589a59e7a59263'
```

```suggestion
    patches: 'ce802bbf81e25f919381e10b0e589a59e7a59263 833ca63be2ab14871874ccb6974921e8952802e9'
```
you replaced the wrong one. The first one was `da-master-8.8.1`. The second one is https://github.com/digital-asset/ghc/tree/da-unit-ids-8.8.1 which has our custom unit ids.